translations/messages-en.properties

```diff
@@ -888,4 +888,8 @@ contact.profile.delivery_code.miscarriage = Miscarriage
 contact.profile.delivery_code.f = Facility birth
 contact.profile.delivery_code.s = Home birth with skilled attendant
 contact.profile.delivery_code.ns = Home birth without skilled attendant
-contact.profile.delivery_code.unknown = Delivery
\ No newline at end of file
+contact.profile.delivery_code.unknown = Delivery
+sync.status.in_progress=Currently syncing…
+sync.status.not_required=All reports synced
```

Comment:
Should this be "reports" or "docs"? I think docs is more technically correct but makes less sense to users maybe?